src/thinblock.cpp

```diff
@@ -1500,7 +1500,7 @@ void AddThinBlockInFlight(CNode *pfrom, uint256 hash)
         std::pair<uint256, CNode::CThinBlockInFlight>(hash, CNode::CThinBlockInFlight()));
 }
 
-void SendXThinBlock(CBlockRef pblock, CNode *pfrom, const CInv &inv)
+void SendXThinBlock(ConstCBlockRef pblock, CNode *pfrom, const CInv &inv)
```

Comment:
this is working cuase `const` reference could be bind to an non `const` object right?

Reply:
You mean when passing in a `CBlockRef`? Yes, to the best of my knowledge :D